src/components/ui/OakRoundIcon/OakRoundIcon.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+
+import {
+  OakBox,
+  OakIcon,
+  OakIconProps,
+  OakImage,
+  OakImageProps,
+} from "@/components/base";
+
+type ImageProps = OakImageProps & {
+  iconName?: undefined;
+};
+
+type IconProps = OakIconProps & {
+  src?: undefined;
+};
+
+export type OakRoundIconProps = IconProps | ImageProps;
+
+/**
+ * A wrapper around `OakIcon` which applies a rounded background.
+ * Supports either an icon name or an image src
+ */
+export const OakRoundIcon = ({
+  $background = "bg-decorative5-main",
+  $borderRadius = "border-radius-circle",
+  $width = "all-spacing-9",
+  $height = "all-spacing-9",
+  $pa = "inner-padding-ssx",
+  ...rest
+}: OakRoundIconProps) => {
+  const icon = rest.iconName ? (
```

Comment:
🤔 I was thinking that there would be cases where the icon itself would not belong in the icon library (I.e a one shot/super bespoke SVG)

In that case it would be useful to be able to point to the `src` of the asset instead of an `iconName` 

I might be a little off course on that assumption in which case we can remove this.